src/EnergyPlus/DataEnvironment.cc

```diff
@@ -128,7 +129,7 @@ namespace DataEnvironment {
 	Real64 PDIFLW; // Luminous efficacy (lum/W) of sky diffuse solar radiation
 	Real64 SkyClearness; // Sky clearness (see subr. DayltgLuminousEfficacy)
 	Real64 SkyBrightness; // Sky brightness (see subr. DayltgLuminousEfficacy)
-	Real64 StdBaroPress( 101325.0 ); // Standard "atmospheric pressure" based on elevation (ASHRAE HOF p6.1)
+	Real64 StdBaroPress( StdPressureSeaLevel ); // Standard "atmospheric pressure" based on elevation (ASHRAE HOF p6.1)
 	Real64 StdRhoAir; // Standard "rho air" set in WeatherManager - based on StdBaroPress
 	Real64 const stdAtmosphericPressure( 101325.0 ); // one atmosphere, sea level standard pressure 101325 Pa
```

Comment:
@nmerket Well, I think I prefer the explicit SeaLevel in the name, even though AtmosphericPressure technically means at sea level.  How many places does this appear in develop?